src/components/profile-settings/form-field-wrapper.tsx

```diff
@@ -0,0 +1,59 @@
+import { LucideIcon } from 'lucide-react'
+import {
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage
+} from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { UseFormReturn } from 'react-hook-form'
+import { ProfileFormValues } from '@/schemas/userSchema'
+// import { ProfileFormValues } from './schema'
+
+interface FormFieldWrapperProps {
+  form: UseFormReturn<ProfileFormValues>
+  name: keyof ProfileFormValues
+  label: string
+  placeholder: string
+  icon: LucideIcon
+  disabled?: boolean
+  type?: string
+}
+
+export default function FormFieldWrapper({
```

Comment:
I am not entirely sure this file is required. Check the event creation form for inspiration.